### DISCUSSION TITLE
Presento mi practica de pasteleria Sam

### DISCUSSION THREAD

**Discussion by JonathanGrimaldo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#243] [General]
Hola 👋🏻  les presento mi pasteleria puse nombres ficticios jeje, acepto criticas y sugerencias, soy más de back pero ya tengo un poco de tiempo moviendole al bootstrap y con las herramientas que vimos al inicio con el MC, fue más sencillo definir el sitio, este es un ejemplo del sitio, por ahora lo dejare así, me faltaron varios detalles :

Lo desplegue en Netlify más rapido, aqui lo pueden ver: 

https://peaceful-babbage-058739.netlify.app/#

**Comment by JuanSalvador73** [comment]
Excelente sitio, muy buen trabajo.

**Comment by davidtoString** [comment]
Te quedó muy bien!!

**Comment by JLuisPrz** [comment]
Se ve genial, pero te falto agregar la parte donde el cliente puede personalizar su pastel con los sabores y adornos.

**Reply by JonathanGrimaldo** [reply]
@JLuisPrz 😄 sii me faltaron alguno detalles, si alcanzo los agrego 😃

**Comment by jesusbeb** [comment]
Excelente!!! Me gusto mucho, pareciera una pagina de un negocio real

**Comment by doguedogue** [comment]
Se ve genial

**Comment by OmarVenturaP** [comment]
Excelente compañero. Me gustó mucho el apartado de desarrollador, excelente implementación.

![Screenshot_20220304-122639.png](https://user-images.githubusercontent.com/99059639/156821017-0a916588-609d-4ffb-aab9-6423965650ec.png)

Por cierto como observación, aparte de lo que los compañeros ya mencionaron, también la sección de pastelero, estaría bien que la maquetas. 

En términos generales te quedó muy chida la página, muy profesional.

**Comment by DiegowJmz** [comment]
Muy muy rifada. Tiene muy buena estetica y el diseño esta super padre.

Espero llegar a ese nivel de diseño.
